twigs/fingerprint.py

```diff
@@ -69,5 +69,22 @@ def get_inventory(args):
             if len(ssh_issues) != 0:
                 asset_data['tags'].append('SSH Audit')
             asset_data['config_issues'] = ssh_issues
+        if args.wordpress == True:
```

Comment:
This should be done exclusively i.e. if wordpress option is given, only run wordpress discovery and nothing else. Would be a good idea to move this into a separate function.